### DISCUSSION TITLE
华师大的 Johnny Chen 和 Julia 社区

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#73] [1️⃣ 新鲜事]
今天收到来自华师大数学科学学院 Johnny 的邮件，连接了一个非常好的 Julia 社区开源爱好者，后面可以请他来给我们分享下 GSoC 和 开源点亮计划的经验~  👋 @johnnychen94

- Johnny 的 GitHub 主页：https://github.com/johnnychen94
- OSPP 活动的总结：https://discourse.juliacn.com/t/topic/5576
- Johnny 在 Julia 中文社区 2021 冬季会议中的报告：
https://www.bilibili.com/video/BV1vY411W7Dw?p=11

**Comment by bifenglin** [comment]
欢迎欢迎！
